doc/tutorials/video/meanshift/meanshift.markdown

```diff
@@ -57,6 +57,14 @@ low light, low light values are discarded using **cv.inRange()** function.
     @include samples/python/tutorial_code/video/meanshift/meanshift.py
 @end_toggle
 
+@add_toggle_java
+-   **Downloadable code**: Click
+    [here](https://github.com/opencv/opencv/tree/3.4/samples/java/tutorial_code/video/meanshift/MeanshiftDemo.java)
+
+-   **Code at glance:**
+    @include  samples/java/tutorial_code/video/meanshift/MeanshiftDemo.java
```

Comment:
I kept the name like that because the following [code](https://github.com/opencv/opencv/tree/master/samples/java/tutorial_code/video/background_subtraction) for background subtraction has its name written like that